### DISCUSSION TITLE
특정 토픽에 대해 원래 있는 컨슈머 그룹에 추가하여  새로운 컨슈머 그룹이 생겼을 때 언제 데이터부터 가져올까?

### DISCUSSION THREAD

**Discussion by sejongjeong** [kyungjunleeme/kafka_study#34] [Q&A]
특정 토픽에 대해 원래 있는 컨슈머 그룹에 추가하여  새로운 컨슈머 그룹이 생겼을 때 언제 데이터부터 가져올까?

**Comment by sejongjeong** [comment]
auto.offset.reset 옵션?

**Comment by kyungjunleeme** [comment]
https://blog.voidmainvoid.net/514?fbclid=IwZXh0bgNhZW0CMTEAAR0QdbTEMASBUp7NqnNwnI0zjoFqm83omdbVmzy4qIhijNsc8dD9VBPKfGA_aem_dI-5NJOhxqs30CHTI1q-dA

이 설정을 말하시는걸까요?
